noxfile.py

```diff
@@ -252,6 +252,41 @@ def compliance_test_20(session):
     )
 
 
+@nox.session()
+def system(session):
+    """Run SQLAlchemy dialect system test suite."""
+
+    # Sanity check: Only run tests if the environment variable is set.
+    if not os.environ.get("GOOGLE_APPLICATION_CREDENTIALS", "") and not os.environ.get(
+        "SPANNER_EMULATOR_HOST", ""
+    ):
+        session.skip(
+            "Credentials or emulator host must be set via environment variable"
+        )
+
+    if os.environ.get("RUN_COMPLIANCE_TESTS", "true") == "false" and not os.environ.get(
+        "SPANNER_EMULATOR_HOST", ""
+    ):
+        session.skip("RUN_COMPLIANCE_TESTS is set to false, skipping")
+
+    session.install(
+        "pytest",
+        "pytest-cov",
+        "pytest-asyncio",
+    )
+
+    session.install("mock")
+    session.install(".[tracing]")
+    session.install("opentelemetry-api==1.27.0")
+    session.install("opentelemetry-sdk==1.27.0")
+    session.install("opentelemetry-instrumentation==0.48b0")
+    session.run("python", "create_test_database.py")
```

Comment:
If my understanding is correct, we cleanup the databases only when a new system test is run, right? Do we want to keep the database around to check for errors? Shouldn't we do a cleanup post the system tests being run as well?

Reply:
Correct, we were only cleaning up during the following run (and then only databases older than 4 hours). I've added a cleanup job that is executed after the system tests have executed.